promitor-agent-resource-discovery/values.yaml

```diff
@@ -12,8 +12,11 @@ image:
   tag:
 
 azureAuthentication:
-  appId:
-  appKey:
```

Comment:
We'll have to be backward compatible here where the new one is favored over the old one; but we shouldn't break the current experience.